pkg/util/sys/device.go

```diff
@@ -43,13 +43,19 @@ const (
 	MultiPath = "mpath"
 	// LinearType is a linear type
 	LinearType = "linear"
-	sgdiskCmd  = "sgdisk"
+	// GPTType is a gpt partition type
+	GPTType = "gpt"
+
 	// CephLVPrefix is the prefix of a LV owned by ceph-volume
 	CephLVPrefix = "ceph--"
 	// DeviceMapperPrefix is the prefix of a LV from the device mapper interface
 	DeviceMapperPrefix = "dm-"
 )
 
+const (
+	sgdiskCmd = "sgdisk"
```

Comment:
why did this move?

Reply:
It's a non-exported constant and not related to the disk types above, so I thought it would be best to move it.